mainapp/models.py

```diff
@@ -74,7 +74,7 @@ class Request(models.Model):
     )
     location = models.CharField(max_length=500,verbose_name='Location - സ്ഥലം')
     requestee = models.CharField(max_length=100,verbose_name='Requestee - അപേക്ഷകന്‍റെ പേര്')
-    requestee_phone = models.CharField(max_length=10,verbose_name='Requestee Phone - അപേക്ഷകന്‍റെ ഫോണ്‍ നമ്പര്‍', validators=[RegexValidator(regex='^[6-9]\d{9}$', message='Please Enter 10 digit mobile number', code='invalid_mobile')])
+    requestee_phone = models.CharField(max_length=13,verbose_name='Requestee Phone - അപേക്ഷകന്‍റെ ഫോണ്‍ നമ്പര്‍', validators=[RegexValidator(regex='^(\+91[\-\s]?)?[0]?(91)?[456789]\d{9}$', message='Please Enter 10 digit mobile number or landline as 0<std code><phone number>', code='invalid_mobile')])
```

Comment:
The new regex will match double 91 as well: `+91-915222239588`. Try the following regex: 
`r'^((\+91|0)[\- ]*)?[456789]\d{9}$'`